snaps/how-to/work-with-existing-snaps.md

```diff
@@ -0,0 +1,31 @@
+# Working with third-party snaps
+
+While some snaps are designed to be one-off and dedicated one and only one Dapp, this isn't always the case. Third-party developers like yourself can use these snaps to build your own Dapps, or even to compose with another snap.
+
+This is possible because [snaps can expose an RPC API](../reference/exports/#onrpcrequest). Snaps can decide to make their API available to other Dapps, other snaps, or both, by requesting the [`endowment:rpc`](../reference/permissions.md#endowmentrpc) permission.
+
+## Installing a snap
+
+Installing a snap is done by using the [`wallet_requestSnaps`](../reference/rpc-api.md#wallet_requestsnaps) method from your Dapp or your snap.
```

Comment:
We shouldn't endorse snaps talking or installing other snaps for the moment

Reply:
`wallet_requestSnaps` is blocked in snaps currently FYI.

Reply:
@FrederikBolding I'm curious what's the reason that it's blocked? and how does that relate to the `snaps: true` in the `endowment:rpc`?